python/lsst/ip/diffim/getTemplate.py

```diff
@@ -126,46 +126,51 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         butlerQC.put(outputs, outputRefs)
 
     def getOverlappingExposures(self, inputs):
-        """Return lists of coadds and their corresponding dataIds that overlap
-        the detector.
+        """Return a data structure containing the coadds that overlap the
+        specified bbox projected onto the sky, and a corresponding data
+        structure of their dataIds.
+        These are the appropriate inputs to this task's `run` method.
 
         The spatial index in the registry has generous padding and often
```

Comment:
I think I don't understand what "spatial index" here is referring to. If this is one of the inputs maybe its worth explaining which?

Reply:
Please have another look at this text: I reworked it a bit to hopefully be more clear?